components/pkg-export-kubernetes/tests/KubernetesManifestTest.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
A bit confused by this test. You have the secret commented out, but you end up using it on 35 of this file. I would get rid of all the mention of secret and the commented out code. Otherwise this manifest file would error out in a k8s cluster as it would expect a secret to be present.

Reply:
Where is the secret commented out in?

Reply:
Right below this comment, the secret creation is commented out.

Reply:
Oh! I didn't realize we are creating a secret in this commented out code. This has been the part of the manifest that was generated already by the first k8s exporter @krnowak wrote and I just kept this as is. I was supposed to ask about it but then I forgot. :(

Anyway, so this part is supposed to create the file that configSecretName later specifies?

Reply:
Yes, exactly.

I would suggest the following test scenarios if I understand correctly:
1. Basic manifest file with no configSecretName and the secret commented out.
2. Bind manifest file.
3. Manifest file that has everything uncommented.

Reply:
Alright, i need to fix the exporter code then, Although we don't want commented out stuff though. If secret is not specified, it's just not there in the manifest.

Reply:
Yep, agreed. 👍  I thought that's how it is already...